src/Containers/Dashboard/Comments/Comments.module.css

```diff
@@ -0,0 +1,45 @@
+.CommentBox{
```

Comment:
css class name never use to be in camel case it must be `comment-box` instead of `CommentBox `